### PR TITLE
Fix the manifest file to include the right helpers things and docs

### PR DESCRIPTION
**Pull request by Cadair**
see sunpy/ndcube#211

### CODE REVIEW

**Comment by nabobalis** [comment]
Need to update the template too.

File: MANIFEST.in [hunk 1]
```diff
@@ -9,5 +9,23 @@ include RELEASE.rst
 include setup.py
 include setup.cfg
 
-recursive-include astropy_helpers *
-exclude astropy_helpers/.*
+# Include the docs and examples
+recursive-include docs *
+recursive-include examples *
+
```
Comment (nabobalis) [inline MANIFEST.in]:
```suggestion

prune build
prune docs/_build
prune docs/api

```
Just in case?

Comment (Cadair) [reply MANIFEST.in]: hehe you noticed that at the same time I did.

**Comment by nabobalis** [comment]
The template repo is actually ok but since riot is broken again I cant tell anyone that.

**Comment by Cadair** [comment]
:man_facepalming: come on matrix.org sort it out.

**Comment by Cadair** [comment]
Wait, if the template is ok, how did NDCube end up in trouble.

**Comment by nabobalis** [comment]
> Wait, if the template is ok, how did NDCube end up in trouble.

A great question to which i have no idea.

**Pull request merged**